sentinel-dashboard/src/main/resources/application.properties

```diff
@@ -12,6 +12,7 @@ logging.pattern.file= %d{yyyy-MM-dd HH:mm:ss} [%thread] %-5level %logger{36} - %
 #auth settings
 auth.filter.exclude-urls=/,/auth/login,/auth/logout,/registry/machine,/version
 auth.filter.exclude-url-suffixes=htm,html,js,css,map,ico,ttf,woff,png
+# If auth.enabled=false, Sentinel console disable login
```

Comment:
The comment is good, suggest remain `auth.enabled=true` after the comment line, in order to make it easy for user to understand and modify.

Reply:
The problem same above